### DISCUSSION TITLE
Bug with `-no:milestone` filter in auto-add to project workflow

### DISCUSSION THREAD

**Discussion by Zimmi48** [community/community#53212] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

When I edit the filter for the auto-add to project workflow, I can end up with `-no:milestone` being part of my filter, just by using auto-complete (for this, I need to type "m", choose "Exclude milestone" in the milestone auto-complete menu, then choose "No milestone" in the new auto-complete menu). This `-no:milestone` filter happens to be a valid query when searching for issues in a repository. Yet, when I try to save the workflow, I get the following error message:
```
Actions query argument contains invalid tokens: no:milestone
```
The issue seems to happen only with `-no:milestone`. In particular, `no:milestone` is correctly supported.

Also, note that I do not get the invalid filter red underline when building the filter, I only get the error message when I save the workflow.

EDIT: this may or may not be related, but in fact the `-no:milestone` filter may not work so well even in issue search, as demonstrated by this query: https://github.com/coq/coq/pulls?q=is%3Apr+-base%3Amaster+-no%3Amilestone

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by saenzramiro** [comment]
This also occurs with `milestone:*` filter.

**Comment by JosXa** [comment]
`created:>date` and `updated:<date` also give an error 🤔

**Comment by SooluThomas** [comment]
`in:title` also errors
